test/endpoints/auth.spec.js

```diff
@@ -19,3 +20,21 @@ describe('Authentication', () => {
     it('Should be able to login', () => should(2).equal(2));
   });
 });
+
+
+describe('POST to api/auth/signup', () => {
+  it('Should return a JWT upon user signup', (done) => {
+    request
+      .post('/api/auth/signup')
+      .send({
+        name: 'stevenson',
+        email: 'simonsinec@gmail.com',
+        password: 'password567'
+      })
+      .end((err, res) => {
+        res.body.message.should.equal('Signed up successfully, please check email for activation link');
```

Comment:
Are you testing that the user receives a JWT or that he receives an activation link? @ddouglasz